lib/zone.ts

```diff
@@ -119,7 +119,7 @@
  * zones are children of the root zone.
  *
  */
-interface Zone {
+export interface Zone {
```

Comment:
This will prevent the Zone from being imported ambiently. Sorry we can't do it that way.